### PR TITLE
Translation Strings

### PR DESCRIPTION
**Pull request by CodeMouse92**
Make proper use of translation strings throughout.

### CODE REVIEW

**Comment by CodeMouse92** [comment]
Handed off. Rebase, modify, and merge at will.

**Comment by CodeMouse92** [comment]
Rebased atop `main` (required force push). Ready for review.

File: Corgibytes.Freshli.Cli/CommandRunners/Cache/DestroyCommandRunner.cs
Comment (mscottford) [inline Corgibytes.Freshli.Cli/CommandRunners/Cache/DestroyCommandRunner.cs:23]: Any of the translation strings that reference a file name should be adjusted so that the placeholder for the value is included in the translated string. This gives translators the flexibility to place words after the value if it is required. But the `{options.CacheDir.FullName}` construct isn't going to work. Instead, we'll need to use [`String.Format`](https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=net-6.0#system-string-format(system-string-system-object)) and specify the placeholder by specifying `{0}` or `{1}`, for example.

**Review by mscottford — CHANGES_REQUESTED** [review]
This looks pretty good. I've just got one small change that I think needs to be made to the translations for strings that include other values.

Also, I don't understand Spanish well enough to comment on how well these translations communicate the original intent as it is expressed in English. For that, I'm just going to trust. :smile:

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:36]:
```suggestion
    <value>Indica el formato del resultado. (distingue entre mayúsculas y minúsculas.)</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:39]:
```suggestion
    <value>Especifica cómo mostrar el resultado. (distingue entre mayúsculas y minúsculas.) Admite múltiples valores.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:51]:
```suggestion
    <value>Omitido</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:54]:
```suggestion
    <value>Existen actualizaciones disponibles.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:60]:
```suggestion
    <value>El comando principal fue ejecutado</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:63]:
```suggestion
    <value>Ejecutando el comando de autenticación.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:72]:
```suggestion
    <value>¿Estas seguro de que quieres eliminar por completo el directorio de cache {0}?</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:75]:
```suggestion
    <value>La operación fue cancelada. El cache no fue eliminado.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:84]:
```suggestion
    <value>El directorio de cache ya fue eliminado o no existe.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:87]:
```suggestion
    <value>El directorio especificado no fue eliminado ya que no es un cache de Freshli.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:57]:
```suggestion
    <value>Ejecutando el comando de escaneo.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:33]:
```suggestion
    <value>Devolución de resultados de las métricas de escaneo para el repositorio local indicado.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:30]:
```suggestion
    <value>Eliminación del directorio de cache sin preguntar por confirmación.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:27]:
```suggestion
    <value>Eliminación del directorio de cache.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:24]:
```suggestion
    <value>Verifica que el directorio de cache exista y contenga una base de datos de cache válida.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:21]:
```suggestion
    <value>Gestión de base de datos y directorio local de cache.</value>
```

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:18]:
```suggestion
    <value>Directorio de almacenamiento de archivos temporales.</value>
```

**Review by mfcorgi — CHANGES_REQUESTED** [review]
I added some suggestions for the spanish translations.

**Review by mscottford — APPROVED** [review]
This looks good pending a final review from @mfcorgi. Once we get that done, feel free to merge this in.

**Comment by CodeMouse92** [comment]
I'll do that after @mairadanielaferrari approves, so I don't have to do that multiple times.

File: Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx
Comment (mfcorgi) [inline Corgibytes.Freshli.Cli/Resources/CliOutput.es.resx:105]:
```suggestion
    <value>No se encontró un repositorio con el hash indicado en el cache. </value>
```

**Review by mfcorgi — APPROVED** [review]

**Pull request merged**
